.gitlab/corona-jobs.yml

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------------------------
+# SUNDIALS Copyright Start
+# Copyright (c) 2002-2021, Lawrence Livermore National Security
+# and Southern Methodist University.
+# All rights reserved.
+#
+# See the top-level LICENSE and NOTICE files for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# SUNDIALS Copyright End
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# HIP
+# ------------------------------------------------------------------------------
+
+# Builds with Hip
+corona_clang_hip:
+  parallel:
+    matrix:
+      - COMPILER_SPEC: clang@15.0.0
+        AMDGPU_TARGET: [gfx906]
+  variables:
+    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double amdgpu_target=${AMDGPU_TARGET} +rocm~mpi"
```

Comment:
Why is mpi disabled?

Reply:
It uses srun to run the tests, which Corona no longer supports. I'm currently trying to swap it over to flux.

Reply:
I thought Corona had aliases setup so slurm commands could still be used but if that's not working, setting adding `-DMPIEXEC_EXECUTABLE='flux mini run'` to the configuration should hopefully be all that's needed

Reply:
Unfortunately, `DMPIEXEC_EXECUTABLE` seems to be for specifying where the scheduler is, so if I do `flux mini run` it gets confused when it tries to look for it.